### PR TITLE
Adding Kotlin toggles

### LINKED ISSUE #180: snippets/examples code switch for changing between Kotlin and Java
Similiar to iOS having both swift and objective-c language variants, Android has Kotlin and Java now and we need to make it easy to switch between examples and snippet languages.

cc: @zugaldia @davidtheclark

### PR DESCRIPTION
**Pull request by langsmith**
Resolves #180 by adding Kotlin toggle code snippets to /android-docs

### CODE REVIEW

**Comment by langsmith** [comment]
@mapbox/navigation-android team:

While adding Kotlin versions of navigation code, it looks like the docs had some outdated Java snippets and explanations. **Please** take a look at the two things listed below and adjust the Java snippets and writing. I will then add Kotlin for the Java that you adjusted.

1. `bearing` doesn't seem to be a callback method that exists anymore? Replace code/writing with a different camera callback as example? https://www.mapbox.com/android-docs/navigation/overview/camera/

screen shot 2018-10-05 at 9 43 02 am

2. The Java code in https://www.mapbox.com/android-docs/navigation/overview/navigation-ui/#step-2

**Comment by danesfeder** [comment]
Thanks for the heads up here @langsmith - @devotaaabel @Guardiola31337 do either of you mind running with a docs update after the `0.21.0` release?

**Comment by devotaaabel** [comment]
@danesfeder @langsmith I'll do it

**Comment by langsmith** [comment]
Great, thanks @devotaaabel . I updated a bunch of code as I set up the toggles, but for some reason, I just couldn't figure out what code was needed to update the sections that I mentioned in this pr's OP ticket.

File: src/pages/maps/overview/query.md [hunk 1]
```diff
@@ -24,28 +26,57 @@ Since the features come from tiled vector data or GeoJSON data that is converted
 
 A common usage for querying the map is to acquire information at a specific position the user is looking at. The point must be viewable inside the user devices viewport and fully rendered before you can access any information. Querying the map only accepts a screen pixel value instead of LatLng so in many cases you'll need to convert beforehand. In the snippet below, the maps clicked on which provides a LatLng we use to query the map and get the properties at that location.
 
-```java
+{{
+<CodeLanguageToggle id="query-features" />
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/query.md]: Looks like you have two toggles with the same `id` which is causing this: 

image

This should be resolved if each `id` on a single page are unique.

File: src/pages/navigation/overview/index.md [hunk 2]
```diff
@@ -64,8 +70,22 @@ If you don't have a Mapbox account, sign up for one [here](https://www.mapbox.co
...
-  }
-```
+}
+`}
+
+kotlin={`
+override fun onCreate(savedInstanceState: Bundle?) {
+	super.onCreate(savedInstanceState)
+
+	val navigation = MapboxNavigation(this, "<access token>")
```
Comment (colleenmcginnis) [inline src/pages/navigation/overview/index.md]: I have a new way to pull in access tokens. I can open a follow-up PR that addresses this throughout the repo.

File: src/pages/maps/overview/query.md (hunk 1 above)
Comment (langsmith) [reply src/pages/maps/overview/query.md]: Fixed.

File: src/pages/navigation/overview/index.md (hunk 2 above)
Comment (langsmith) [reply src/pages/navigation/overview/index.md]: Ticketed at https://github.com/mapbox/android-docs/issues/662

**Pull request merged**